tests/test_floss.py

```diff
@@ -411,3 +429,91 @@ def test_aamp_floss_inf_nan(substitute, substitution_locations):
             npt.assert_almost_equal(ref_P, comp_P)
             npt.assert_almost_equal(ref_I, comp_I)
             npt.assert_almost_equal(ref_T, comp_T)
+            npt.assert_almost_equal(ref_mp[:, 3], stream._mp[:, 3])
```

Comment:
I added this test but I now think it might not be needed. what do you think @seanlaw ?